### PR TITLE
feat: added input component

### PR DESCRIPTION
**Pull request by TejasShekar**
### Added:
1. Simple Input
2. Input with validation styles

### CODE REVIEW

File: Components/Input/input.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            &lt;div class=&quot;input-container gap-0&quot;&gt;
+             &lt;label class=&quot;input-label&quot; for=&quot;text-input&quot;&gt;Username&lt;/label&gt;
+             &lt;input
+              id=&quot;text-input&quot;
+              type=&quot;text&quot;
+              placeholder=&quot;Enter your username here&quot;
+              /&gt;
+            &lt;/div&gt;
+            </code>
+            <!-- <button id="button" class="snippet-btn"><i class="fas fa-copy"></i></button> -->
```
Comment (bharati-21) [inline Components/Input/input.html:76]: You can remove the commented out code. 👍🏽

File: Components/Input/input.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+              type=&quot;password&quot;
+              placeholder=&quot;Enter your password here&quot;
+            /&gt;
+            &lt;div class=&quot;error-message&quot;&gt;&lt;i class=&quot;fas fa-exclamation-triangle&quot;&gt;&lt;/i&gt; Wrong Password !&lt;/div&gt;
+          &lt;/div&gt;
+              </code>
+              <!-- <button id="button" class="snippet-btn"><i class="fas fa-copy"></i></button> -->
+          </pre>
+        </section>
+        <!-- <div id="copiedSnackbar" class="">Copied!</div> -->
```
Comment (bharati-21) [inline Components/Input/input.html:137]: And here.

File: Components/Input/input.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            &lt;label class=&quot;input-label&quot; for=&quot;error-input&quot;&gt;Error Validation&lt;/label&gt;
+            &lt;input
+              id=&quot;error-input&quot;
+              type=&quot;password&quot;
+              placeholder=&quot;Enter your password here&quot;
+            /&gt;
+            &lt;div class=&quot;error-message&quot;&gt;&lt;i class=&quot;fas fa-exclamation-triangle&quot;&gt;&lt;/i&gt; Wrong Password !&lt;/div&gt;
+          &lt;/div&gt;
+              </code>
+              <!-- <button id="button" class="snippet-btn"><i class="fas fa-copy"></i></button> -->
```
Comment (bharati-21) [inline Components/Input/input.html:134]: Same suggestion here as well.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

**Review by BelideNikhil — APPROVED** [review]
Use font weight and border radius as variables if possible

File: Components/Input/input.html (hunk 1 above)
Comment (TejasShekar) [reply Components/Input/input.html:76]: I've left them there as I plan to fix the code block in future.

File: Components/Input/input.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+#success-input {
+  border: 3px solid var(--success);
+}
+
+#error-input {
+  border: 3px solid var(--danger);
+}
+
+.gap-0 {
```
Comment (MaruthiPothuganti) [inline Components/Input/input.css:34]: This can be moved into utility class

**Review by MaruthiPothuganti — CHANGES_REQUESTED** [review]
Suggested a change. Everything else looks good.

File: Components/Input/input.css (hunk 4 above)
Comment (TejasShekar) [reply Components/Input/input.css:34]: I currently have not created utility classes yet. Will do that.

**Pull request merged**